### PR TITLE
Add 20 index files to 1.12 Chinese docs

### PR DESCRIPTION
**Pull request by idealhack**

### CODE REVIEW

**Comment by idealhack** [comment]
This PR is just a cherry-pick and rebase of the following PRs for better reviewing:

- https://github.com/kubernetes/website/pull/11116
- https://github.com/kubernetes/website/pull/11117
- https://github.com/kubernetes/website/pull/11118
- https://github.com/kubernetes/website/pull/11119
- https://github.com/kubernetes/website/pull/11120
- https://github.com/kubernetes/website/pull/11121
- https://github.com/kubernetes/website/pull/11122
- https://github.com/kubernetes/website/pull/11123
- https://github.com/kubernetes/website/pull/11124
- https://github.com/kubernetes/website/pull/11125
- https://github.com/kubernetes/website/pull/11126
- https://github.com/kubernetes/website/pull/11127
- https://github.com/kubernetes/website/pull/11128
- https://github.com/kubernetes/website/pull/11129
- https://github.com/kubernetes/website/pull/11130
- https://github.com/kubernetes/website/pull/11131
- https://github.com/kubernetes/website/pull/11132
- https://github.com/kubernetes/website/pull/11133
- https://github.com/kubernetes/website/pull/11134
- https://github.com/kubernetes/website/pull/11135

File: content/zh/docs/tasks/job/_index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!--
+---
+title: "Run Jobs"
+weight: 50
+---
+-->
+
+---
+title: "运行任务"
```
Comment (tengqm) [inline content/zh/docs/tasks/job/_index.md]: 建议 Jobs 不翻译

**Comment by yue9944882** [comment]
👍🏻

**Pull request merged**